src/dapp/Blockchain.ts

```diff
@@ -372,6 +374,52 @@ export class BlockChain {
     };
   }
 
+  private async getExploringAccount(exploringAccount): Promise<Account> {
+    if (!this.web3 || this.isTrial) {
+      return {
+        farm: [
+          {
+            createdAt: 0,
+            fruit: Fruit.None,
+          },
+          {
+            createdAt: 0,
+            fruit: Fruit.Sunflower,
+          },
+          {
+            createdAt: 0,
+            fruit: Fruit.Sunflower,
+          },
+          {
+            createdAt: 0,
+            fruit: Fruit.Sunflower,
+          },
+          {
+            createdAt: 0,
+            fruit: Fruit.None,
+          },
+        ],
+        balance: 0,
+        id: exploringAccount,
+      };
+    }
+
+    const farm = await this.alchemyFarm.methods
```

Comment:
I'm curious if we may be able to simplify the approach. Instead of creating 2 new methods in Blockchain.ts,  could we instead update `this.account` in Blockchain.ts to point to the new farm address?

If we did this, we could just reload the farm and it should pull in the correct values.